### PR TITLE
Support query parameter `visible_to_repository` in ListOrganizationRunnerGroups

### PR DESCRIPTION
**Pull request by TingluoHuang**
Fixing https://github.com/google/go-github/issues/2328 

Support `visible_to_repository` query parameter in `ListOrganizationRunnerGroups`.

### CODE REVIEW

File: github/actions_runner_groups.go
Comment (gmlewis) [inline github/actions_runner_groups.go:64]:
```suggestion
// ListOrgRunnerGroupOptions extend ListOptions to have the optional parameters VisibleToRepository.
```

File: github/actions_runner_groups.go
Comment (gmlewis) [inline github/actions_runner_groups.go:84]: Instead of supporting two methods with identical functionality but with an added field,
let's please get rid of this line entirely and go ahead and break the API by just changing line 73 above, as shown.

File: github/actions_runner_groups.go
Comment (gmlewis) [inline github/actions_runner_groups.go:73]:
```suggestion
func (s *ActionsService) ListOrganizationRunnerGroups(ctx context.Context, org string, opts *ListOrgRunnerGroupOptions) (*RunnerGroups, *Response, error) {
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @TingluoHuang !

File: github/actions_runner_groups.go
Comment (TingluoHuang) [reply github/actions_runner_groups.go]: Updated, thanks.

**Review by gmlewis — APPROVED** [review]
Thank you, @TingluoHuang !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by raynigon — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
> LGTM

Thank you, @raynigon !
Merging.

**Pull request merged**
